packages/playground/src/router/index.ts

```diff
@@ -492,7 +492,7 @@ const router = createRouter({
     {
       path: "/minting",
       component: () => import("../views/minting_view.vue"),
-      meta: { title: "Minting" },
+      meta: { title: "Minting", info: { page: "info/minting.md" } },
```

Comment:
Resolved since this issue appeared in other solutions and a separate issue was created for it. https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1600